skimage/transform/_geometric.py

```diff
@@ -805,7 +805,12 @@ def __init__(self, matrix=None, scale=None, rotation=None, shear=None,
             if translation is None:
                 translation = (0, 0)
 
-            sx, sy = scale
+            scale = np.atleast_1d(scale)
+            if len(scale) == 1:
+                sx = sy = scale
+            else:
+                sx, sy = scale
```

Comment:
This is fine, but I would save a line by doing:

```suggestion
            if np.isscalar(scale):
                sx = sy = scale
            else:
                sx, sy = scale
```